test_problems/rankine_democxx/rankine.cpp

```diff
@@ -70,7 +65,7 @@ int openRankine(int np, void* p)
     double heat_in = h["3"] - h["2"];
     double efficiency = work/heat_in;
 
-    cout << "efficiency = " << efficiency << endl;
+    printf("efficiency = %8.6g", efficiency);
```

Comment:
If you don't want to use `cout`, I'd recommend Cantera's `writelog` function over `printf`.